modules/bundle/hybrid_programming/scripts/inference.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence, Union
+
+import torch
+from monai.bundle import ConfigParser
+from monai.data import decollate_batch
+from monai.utils.enums import CommonKeys
+
+
+def run(config_file: Union[str, Sequence[str]], ckpt_path: str):
```

Comment:
perhaps include the `logging.conf` if it's useful?

Reply:
Hi @wyli ,

`logging.conf` is not useful in this example.
The engines of spleen segmentation bundle can work with `logging.conf`, this example doesn't use engines.

Thanks.